src/System.IO.Ports/tests/Legacy/SerialPort/ReadTimeout.cs

```diff
@@ -36,146 +36,167 @@ private enum ThrowAt { Set, Open };
 
     #region Test Cases
 
+    [ActiveIssue(15961)]
     [ConditionalFact(nameof(HasLoopbackOrNullModem))]
     public void ReadTimeout_Default_Read_byte_int_int()
```

Comment:
I'd like to try grabbing them in chunks for now. Things that are using StopWatch and thread priority are a pretty rational bucket I think.